src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx

```diff
@@ -19,7 +18,6 @@ export class ConfirmationPage extends React.Component {
   componentDidMount() {
     focusElement('.schemaform-title > h1');
     scrollToTop();
```

Comment:
You can use our reusable module for this:
```js
import scrollToTop from 'platform/utilities/ui/scrollToTop';
```

Reply:
Good catch, I actually need to remove this